recipes/IWSLT22_lowresource/train.py

```diff
@@ -0,0 +1,420 @@
+    elif hparams["sorting"] == "descending":
+        # use smaller dataset to debug the model
+        if hparams["debug"]:
+            datasets["train"] = datasets["train"].filtered_sorted(
+                key_min_value={"duration": hparams["sorting_min_duration"]},
+                key_max_value={"duration": hparams["sorting_max_duration"]},
+                sort_key="duration",
+                reverse=True,
+            )
+            datasets["valid"] = datasets["valid"].filtered_sorted(
+                key_min_value={"duration": hparams["sorting_min_duration"]},
+                key_max_value={"duration": hparams["sorting_max_duration"]},
+                sort_key="duration",
+                reverse=True,
+            )
+        else:
+            datasets["train"] = datasets["train"].filtered_sorted(
+                sort_key="duration", reverse=True
+            )
+            datasets["valid"] = datasets["valid"].filtered_sorted(
+                sort_key="duration", reverse=True
+            )
+
+        hparams["dataloader_options"]["shuffle"] = False
+        hparams["dataloader_options"]["shuffle"] = False
+    elif hparams["sorting"] == "random":
+        # use smaller dataset to debug the model
+        if hparams["debug"]:
+            datasets["train"] = datasets["train"].filtered_sorted(
+                key_min_value={"duration": 3},
+                key_max_value={"duration": hparams["sorting_max_duration"]},
+                sort_key="duration",
+            )
+            datasets["valid"] = datasets["valid"].filtered_sorted(
+                key_min_value={"duration": hparams["sorting_min_duration"]},
+                key_max_value={"duration": hparams["sorting_max_duration"]},
+                sort_key="duration",
+            )
+
+        hparams["dataloader_options"]["shuffle"] = True
+    else:
+        raise NotImplementedError(
+            "sorting must be random, ascending or descending"
+        )
+
+    return datasets, tokenizer
+
+
+if __name__ == "__main__":
+
+    # Load hyperparameters file with command-line overrides
+    hparams_file, run_opts, overrides = sb.parse_arguments(sys.argv[1:])
+    with open(hparams_file) as fin:
+        hparams = load_hyperpyyaml(fin, overrides)
+
+    # creates a logger
+    logger = logging.getLogger(__name__)
+
+    # If distributed_launch=True then
+    # create ddp_group with the right communication protocol
+    sb.utils.distributed.ddp_init_group(run_opts)
+
+    # Create experiment directory
+    sb.create_experiment_directory(
+        experiment_directory=hparams["output_folder"],
+        hyperparams_to_save=hparams_file,
+        overrides=overrides,
+    )
+
+    # Create main experiment class
+    st_brain = ST(
+        modules=hparams["modules"],
+        hparams=hparams,
+        run_opts=run_opts,
+        checkpointer=hparams["checkpointer"],
+    )
+
+    # Data preparation
+    from recipes.IWSLT22_lowresource import prepare_iwslt22
```

Comment:
Can it be loaded directly ?
e.g. `import prepare_iwslt22`

I did not test it - it's just to have it similar to other recipes as in:
https://github.com/mzboito/speechbrain/blob/6519b160d9fe1e9e20c0bd71fded36827cc73609/recipes/LibriSpeech/ASR/seq2seq/train.py#L355